From 8d87974a34c055e0daee3a83f11c539fc98fd6bf Mon Sep 17 00:00:00 2001
From: Eric Garver <eric@garver.life>
Date: Wed, 26 Jun 2024 15:31:38 -0400
Subject: [PATCH 22/22] v2.2.0: chore(service): remove Conflicts with nftables

Now that firewalld uses the table owner flag it can coexist with the
nftables service.

(cherry picked from commit 9c6cb982981ad0aee5b85773823614ca7fd69073)
---
 config/firewalld.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/firewalld.service.in b/config/firewalld.service.in
index afbe0ac5def7..b757a08f28dc 100644
--- a/config/firewalld.service.in
+++ b/config/firewalld.service.in
@@ -4,7 +4,7 @@ Before=network-pre.target
 Wants=network-pre.target
 After=dbus.service
 After=polkit.service
-Conflicts=iptables.service ip6tables.service ebtables.service ipset.service nftables.service
+Conflicts=iptables.service ip6tables.service ebtables.service ipset.service
 Documentation=man:firewalld(1)
 
 [Service]
-- 
2.43.5

